rock_gazebo.orogen

```diff
@@ -49,10 +49,17 @@ task_context "ModelTask" do
     # The covariance matrix to be used as velocity covariance. Default is unset (NaN)
     property 'cov_velocity', '/base/Matrix3d'
 
+    # The acceleration of the model w.r.t. the world frame
+    output_port 'ang_accel_samples', '/base/samples/RigidBodyAcceleration'
```

Comment:
`ang` is really not that used as a shorthand for angular, make `angular` explicit. `acceleration` is sometimes shortened to `acc` in Rock, but then `angular_acc_samples` really looks weird. What about the explicit `angular_acceleration_samples` ?